README.md

```diff
@@ -129,7 +130,10 @@ I would like to thank my coding partners that helped me alot for this project.
 
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you add some more meaningful features in the Key Feature section to make your readme descriptive 👍